site/frontend/src/pages/dashboard.ts

```diff
@@ -92,7 +92,7 @@ function renderRuntime(element: string, data: [number], versions: [string]) {
       text: `Average time for a runtime benchmark`,
     },
     yAxis: {
-      title: {text: "Seconds"},
+      title: {text: "Nanoseconds"},
```

Comment:
I would actually prefer to just show a larger unit (milliseconds?), and then multiply the values, rather than showing these super big numbers.